docs/tutorials/analysis/1D/cta_sensitivity.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "id": "fbb1c19b",
```

Comment:
Is there anything we can do about these "random" meta updates, that just depend on the `jupyter` versions?

Reply:
@adonath 

You can also see in this PR that the `nbformat` version metadata have been modified for all notebooks to the values below.

```
  "nbformat": 4,
  "nbformat_minor": 4
```

The "id" tags in the cells are a consequence of the upgrading of `nbformat` in the notebooks. Since we unpinned the version of `nbformat` in the `environement-dev.yaml` file [here](https://github.com/gammapy/gammapy/pull/3773), this upgrading of notebooks is needed if we want to avoid warnings in the step of notebooks manipulation (adding box for links and binder, create link to API ref, etc.),.

It is completely harmless.
These ids and their values will not change from one doc building to the next if the notebooks are not modified.

Reply:
OK, thanks for the explanation. My biggest worry here is that the diffs become hard to review and long-term we might blow up the size of the `.git` folder with these updated which are basically unrelated to the actual content. I guess the only solution would to change to a text-based notebook format...

Reply:
This is not a problem.
The id tags and values do not change when the notebook are re-executed. But if we add a new cell, this one will be added with an id tag.

Reply:
But if not all developers are on the same `nbformat` version, another PR will remove them again, so we should definitely make sure this is not happening. All in all this is not ideal...I hope we can change to a text based format in future.

Reply:
Yes.

Anyhow, we have a development environment `environment-dev.yml` with the version of the `nbformat` dependency unpinned. This is also the environment of the CI, developers working with old versions of `nbformat` would not comply with this development environment and modifs in the metadata of the notebeook would show up in the PR.

The other option would we to remove the id tags and pin the version of `nbformat`